lib/index.js

```diff
@@ -10,9 +10,51 @@ const Joi = require('joi')
  * @param {object} route.response - The Joi's schema for the response
  * @param {object} options - Joi's options
  */
-module.exports = (route, options) => {
+module.exports = (route, options, formatter) => {
   // This function will be the actual route
   return (req, res, next) => {
+    /**
+     * Optional - Users can make basic changes to the error message.
+     * The eval() command is run against the formatter string in the following way:
+     * message = eval(`err.message.${formatter}`)
+     * details[].message = eval(`details[].message.${formatter}`)
+     *
+     * This is useful if you want to eliminiate the double quotes
+     * returned by Joi (ex: formatter = 'replace(/"/g, "")')
+     * @param  {Error} err A Joi validation Error
+     * @return {Object}     An object of the form:
+     * {
+     *    message: 'a message',
+     *    details: [{
+     *      // joi validation error details
+     *    }]
+     * }
+     */
+    function formatError (err) {
+      let message = err.message || ''
+      let details = err.details || []
+
+      if (formatter) {
+        try {
+          const msgCmd = `err.message.${formatter}`
+          const detailsCmd = `detail.message.${formatter}`
+
+          message = eval(msgCmd)
```

Comment:
It would be better to have a pointer to a function (to format for example) instead of an eval. (You don't want to call any `eval` function within a query error formatter).
Let's say instead of having `formatError` and `formatter` why not having `formatFn` with a `formatDummyFn`. 
The default formatter would be the `formatDummyFn` but if `formatFn` has been defined then it would be used instead.